sql/telemetry/nondesktop_clients_last_seen_v1/view.sql

```diff
@@ -83,41 +36,138 @@ glean_union AS (
     'VR Browser Baseline' AS app_name,
   FROM
     `moz-fx-data-shared-prod.org_mozilla_vrbrowser.baseline_clients_last_seen`
-)
-SELECT
-  submission_date,
-  client_id,
-  days_seen_bits,
-  days_since_seen,
-  days_created_profile_bits,
-  days_since_created_profile,
-  app_name,
-  os,
-  osversion AS os_version,
-  normalized_channel,
-  campaign,
-  country,
-  locale,
-  distribution_id,
-  metadata_app_version AS app_version
-FROM
-  `moz-fx-data-shared-prod.telemetry.core_clients_last_seen`
-UNION ALL
+),
+-- Within this view, we present a `product` field that is intended to be
+-- short, readable, unambiguous, and reasonably close to the "official" names
+-- of the applications. The app_name field is a messy artifact of raw pings
+-- that we include only to help with debugging.
+-- For Android, we use the internal names "Fennec" and "Fenix" as the product name
+-- to disambiguate between the legacy and current implementations of Firefox for Android.
+--
+-- Expect the following mappings:
+--
+--   "Official" name               | product          | app_name
```

Comment:
I really like this. Will this table live on dtmo or as a udf? Specifically asking with a focus on the official name. Would be a nice to have for it to be in a handy spot.

Reply:
I've refactored this logic to a new UDF `mozfun.norm.product_info` in this PR. That will produce the table in rendered documentation and be consistent with the implementation of the function.